examples/form/mui/useDrawerForm/src/App.tsx

```diff
@@ -18,6 +20,10 @@ const App: React.FC = () => {
     return (
         <ThemeProvider theme={LightTheme}>
             <RefineSnackbarProvider>
+                <CssBaseline />
+                <GlobalStyles
+                    styles={{ html: { WebkitFontSmoothing: "auto" } }}
+                />
```

Comment:
```suggestion
        <ThemeProvider theme={LightTheme}>
            <CssBaseline />
              <GlobalStyles
                  styles={{ html: { WebkitFontSmoothing: "auto" } }}
              />
                <RefineSnackbarProvider>
```

I think `RefineSnackbarProvider` can also benefit the global styles and the baseline. 🤔

Reply:
ups :/